### PR TITLE
Fixing a typo

### PR DESCRIPTION
**Pull request by sfshaza2**
Fixes issue https://github.com/flutter/website/issues/7586

### CODE REVIEW

**Comment by piedcipher** [comment]
Is it better to add `english_words: ^4.0.0` over `english_words: 4.0.0`?

**Comment by sfshaza2** [comment]
Good point, @piedcipher!

File: src/get-started/codelab.md [hunk 1]
```diff
@@ -230,7 +230,7 @@ for a Flutter app.
 
  1. Add the `english_words` package to your project as follows:
 
-    In your IDE, add the line `english_words 4.0.0`
+    In your IDE, add the line `english_words: ^4.0.0`
     just after the `cupertino_icons 1.0.4` line
```
Comment (piedcipher) [inline src/get-started/codelab.md:234]: may be we could add `cupertino_icons: ^1.0.5` or `cupertino_icons: ^1.0.4`. it's fine though

**Review by domesticmouse — APPROVED** [review]

File: src/get-started/codelab.md (hunk 1 above)
Comment (sfshaza2) [reply src/get-started/codelab.md:234]: Yeah, no, I'd rather leave it more general.

Comment (piedcipher) [reply src/get-started/codelab.md:234]: Coool

**Pull request merged**